lib/logger/utils.ts

```diff
@@ -206,3 +206,42 @@ export function withSanitizer(streamConfig: bunyan.Stream): bunyan.Stream {
 
   throw new Error("Missing 'stream' or 'path' for bunyan stream");
 }
+
+/**
+ * A function that terminates exeution if the log level that was entered is
+ *  not a valid value for the bunyan logger.
```

Comment:
```suggestion
 *  not a valid value for the Bunyan logger.
```

Bunyan is a proper noun, so should be capitalized.